scripts/plant_life_hist_eu.py

```diff
@@ -68,7 +68,7 @@ def download(self, engine=None, debug=False):
             new_data.close()
             self.engine.auto_create_table(Table(key,
                                                 cleanup=Cleanup(correct_invalid_value,
-                                                                nulls=[-999.9])),filename=str("new" + key))
+                                                                missingValues=[-999.9])),filename=str("new" + key))
```

Comment:
This maybe a possibility. I will commit the changes for handling the use of missingValues for version < 2.0.0